src/RoadRegistry.BackOffice.ZipArchiveWriters/ExtractHost/GradeSeparatedJunctionArchiveWriter.cs

```diff
@@ -28,10 +29,8 @@ public GradeSeparatedJunctionArchiveWriter(RecyclableMemoryStreamManager manager
         if (request == null) throw new ArgumentNullException(nameof(request));
         if (context == null) throw new ArgumentNullException(nameof(context));
 
-        var junctions =
-            await context.GradeSeparatedJunctions
-                .InsideContour(request.Contour)
-                .ToListAsync(cancellationToken);
+        var junctions = await context.GradeSeparatedJunctions.ToListWithPolygonials(request.Contour, (dbSet, polygon) => dbSet.InsideContour(polygon), x => x.Id, cancellationToken);
```

Comment:
use a line-separated method chain (like what it was)

Reply:
Done